doc/guides/advanced_tutorials/porting_boards.md

```diff
@@ -336,6 +336,125 @@ static const timer_conf_t timer_config[] = {
 /** @} */
 ```
 
+## New Style Common Code
```

Comment:
Does it maybe make sense to move all common code related titles to h3/### and either make `## Using Common Code` the parent or add a new parent heading of like `## Common Code` above that?

It feels a bit weird that using, new style & old style are three different distinct entries in the navigation